src/webots/app/WbProtoCachedInfo.cpp

```diff
@@ -193,7 +193,7 @@ WbProtoCachedInfo *WbProtoCachedInfo::computeInfo(const QString &protoFileName)
   tokenizer.rewind();
   while (tokenizer.hasMoreTokens()) {
     WbToken *token = tokenizer.nextToken();
-    if (token->isIdentifier() && WbNodeUtilities::isDeviceTypeName(token->word())) {
+    if (token->isIdentifier() && WbNodeUtilities::isDeviceTypeName(token->word()) && token->word() != "Connector") {
       protoInfo->mContainsDevices = true;
```

Comment:
Just a minor note: maybe we should change the member and getter name to be more consistent with the code and instead of `mContainsDevice` change it to something like `mNeedsRobotAncestor`.

Reply:
Good point. Are we allowed to change this string in the proto cache file as well or it is not good to do this in a minor version?

Reply:
I think it would be better to do it in a major version.
If I'm not wrong the only way to force an automatic update of the Proto Cache version if the PROTO file doesn't change is to increase the Proto Cache version.
if we do not increate the version, the cache of users' PROTO nodes will contain a wrong value.

Reply:
Another solution will be to support both old and new label in the revision and change it late. but it seems too much work for a small detail.

Reply:
Ok, I will then change only the getter and variable name on the Webots side. and then fix this in develop once merged.